### PR TITLE
Temporarily exclude cent7 machines for sanity.external

### PR DESCRIPTION
**Pull request by LongyuZhang**
- Sanity.external test will run on Rhel and Ubuntu machines
- Cent machines have different docker versions and caused failure
- Related Issue: runtimes/backlog/1383

### CODE REVIEW

**Comment by LongyuZhang** [comment]
Simulated Test Scenario with `if (JOB_NAME.contains ("sanity.external") || JOB_NAME.contains ("Grinder"))`,
The in Grinder, the machine label is `16:19:54  LABEL: ci.role.test&&hw.arch.x86&&sw.os.linux&&!sw.os.cent.7` (hyc_grinder 39202)

**Review by llxia — APPROVED** [review]

File: buildenv/jenkins/openjdk_tests [hunk 1]
```diff
@@ -297,9 +297,9 @@ timestamps{
                 env.EXTRA_OPTIONS += " -Dsemeru.fips=true -Dsemeru.customprofile=${opts}"
             }
 
-            // Temporarily exclude ubuntu 22 machines for criu sanity.external pipeline
-            if (JOB_NAME.contains ("sanity.external") && JOB_NAME.contains("_criu")) {
-                LABEL += "&&!sw.os.ubuntu.22"
+            // Temporarily exclude Cent machines for sanity.external pipeline due to docker issue
```
Comment (karianna) [inline buildenv/jenkins/openjdk_tests:300]:
```suggestion
            // Temporarily exclude CentOS machines for sanity.external pipeline due to docker issue
```

**Review by karianna — APPROVED** [review]

**Comment by karianna** [comment]
Do we have a separate issue to track reversing this?

**Comment by llxia** [comment]
@LongyuZhang could you back port the change in https://github.com/adoptium/aqa-tests/tree/v1.0.1-release? Thanks

**Pull request merged**